MvvmHelpers.UnitTests/WeakEventManagerTests.cs

```diff
@@ -0,0 +1,224 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace MvvmHelpers.UnitTests
+{
+	[TestClass]
+	public class WeakEventManagerTests
+	{
+		static int count;
```

Comment:
I usually use Xunit which runs tests randomly and in parallel therefore seeing `static` in a unit test class always triggers a warning in my head. The comment is here just in case the behavior is similar and could cause breaking tests.